src/Cell.cpp

```diff
@@ -794,81 +940,6 @@ void Cell::addNeighborCell(Cell* cell) {
 
 
 /**
- * @brief Finds and stores a bounding box for the entire geometry.
- */
-void Cell::findBoundingBox() {
-
-  /* Set a default bounding box around the Cell */
-  _min_x = -std::numeric_limits<double>::infinity();
-  _max_x = std::numeric_limits<double>::infinity();
-  _min_y = -std::numeric_limits<double>::infinity();
-  _max_y = std::numeric_limits<double>::infinity();
-  _min_z = -std::numeric_limits<double>::infinity();
-  _max_z = std::numeric_limits<double>::infinity();
-
-  /* Loop over all Surfaces inside the Cell */
-  std::map<int, surface_halfspace*>::iterator iter;
-  Surface* surface;
-  int halfspace;
-  double min_x, max_x, min_y, max_y, min_z, max_z;
-
-  for (iter = _surfaces.begin(); iter != _surfaces.end(); ++iter) {
-
-    surface = iter->second->_surface;
-    halfspace = iter->second->_halfspace;
-
-    max_x = surface->getMaxX(halfspace);
-    max_y = surface->getMaxY(halfspace);
-    max_z = surface->getMaxZ(halfspace);
-
-    min_x = surface->getMinX(halfspace);
-    min_y = surface->getMinY(halfspace);
-    min_z = surface->getMinZ(halfspace);
-
-    if (max_x != std::numeric_limits<double>::infinity() && max_x < _max_x) {
-      _max_x = max_x;
-      _max_x_bc = surface->getBoundaryType();
-    }
-    if (max_y != std::numeric_limits<double>::infinity() && max_y < _max_y) {
-      _max_y = max_y;
-      _max_y_bc = surface->getBoundaryType();
-    }
-    if (max_z != std::numeric_limits<double>::infinity() && max_z < _max_z) {
-      _max_z = max_z;
-    }
-
-    if (min_x != -std::numeric_limits<double>::infinity() && min_x > _min_x) {
-      _min_x = min_x;
-      _min_x_bc = surface->getBoundaryType();
-    }
-    if (min_y != -std::numeric_limits<double>::infinity() && min_y > _min_y) {
-      _min_y = min_y;
-      _min_y_bc = surface->getBoundaryType();
-    }
-    if (min_z != -std::numeric_limits<double>::infinity() && min_z > _min_z) {
-      _min_z = min_z;
-    }
-  }
-
-  /* If we could not find a bounds for any dimension, readjust
-   * it to +/- infinity */
-  if (_max_x == -std::numeric_limits<double>::infinity())
```

Comment:
It looks to me like these readjustments are unnecessary. `_max_x` is initialized to positive infinity and I don't see how a surface could have a maximum x value of negative infinity.